47525 E-F Ink Panel, warnings

tyndmar opened the pull request:
Program changed:
ce/v-est3.w
added EXCLUSIVE option to FOR EACHes that update fields.

Commit:
47525 E-F Ink Panel, warnings
Program changed:
ce/v-est3.w
added EXCLUSIVE option to FOR EACHes that update fields.

brad-advantzware reviewed (changes requested):
The block with For each b-ef of eb that was changed from a NO-LOCK to EXCLUSIVE should likely be completely deleted.  This looks like the purpose was for reftable manipulation and since we are working with native db fields, I don't think this serves a purpose (other than to cause the NO-LOCK error).

Commit:
47525 E-F Ink Panel, warnings
File changed: ce/v-est3.w
Cleaned code
Removed section that tried to make all blanks of a form have same ink unit and side values
